services/housetables/src/main/java/com/linkedin/openhouse/housetables/services/UserTablesServiceImpl.java

```diff
@@ -140,6 +140,11 @@ public void renameUserTable(
     }
     // Renames user table within the same database
     try {
+      log.info(
+          "Renaming user table from {} to {} in database {}",
+          fromTableId,
+          toTableId,
+          fromDatabaseId);
       htsJdbcRepository.renameTableId(fromDatabaseId, fromTableId, toTableId, metadataLocation);
```

Comment:
Are we not passing the `toDatabaseId` here although database is same?

Reply:
I can add the toDatabaseId so it's easier to extend functionality in the future, yes they'd be the same so it'd be a noop